### PR TITLE
made command to move Lift up at beginning of Auton

### LINKED ISSUE #117: Automatically lift Lift up at beginning of auton
add new command to lift lift up
add to beginning of auton

### PR DESCRIPTION
**Pull request by Griffin9881**
### Pull Request

---------------------------------------------

#### Issue Number

 Closes #117

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 1]
```diff
@@ -140,6 +140,10 @@ public void runRightUp() {
         }
     }
 
+    public boolean getLiftEncoder() {
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Lift.java:143]: The name of this function doesn’t really indicate what it does. isLiftAtTop would be more descriptive. Also we need a separate one for right and left

File: src/main/java/frc/robot/commands/FullLiftCommand.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    @Override
+    public void execute() {
+        lift.runLeftUp();
+        lift.runRightUp();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return !lift.getLiftEncoder();
```
Comment (reediculous456) [inline src/main/java/frc/robot/commands/FullLiftCommand.java:27]: This command is only different from the existing ones in the finish condition and I would say the current up command could use the same finish condition so I would just make changes to the existing command

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -68,6 +71,7 @@ public void disabledExit() {
 
     @Override
     public void autonomousInit() {
+        fullLiftCommand.schedule();
```
Comment (reediculous456) [inline src/main/java/frc/robot/Robot.java:74]: Since we don’t need to reference this, I would just do new LiftCommand().schedule()

**Review by reediculous456 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 3 above)
Comment (Griffin9881) [reply src/main/java/frc/robot/Robot.java:74]: i made the change and the pull request doesnt show it and it sad

**Review by LandisCrawford6114 — DISMISSED** [review]

**Review by LandisCrawford6114 — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 4]
```diff
@@ -140,6 +140,10 @@ public void runRightUp() {
         }
     }
 
+    public boolean isLiftUp() {
+        return rightLiftEncoder.getPosition() < LiftConstants.maxExtension;
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Lift.java:144]: This still needs adjusted to account for both sides

**Pull request merged**